src/components/InventoryTable/EntityTableToolbar.test.js

```diff
@@ -207,10 +229,14 @@ describe('EntityTableToolbar', () => {
         });
 
         describe('delete filter', () => {
+            beforeEach(() => {
+                mockTags.onGet(/\/api\/inventory\/v1.*/).reply(200, { results: [] });
+                mockSystemProfile.onGet().reply(200, { results: [] });
+            });
+
             it('should dispatch action on delete filter', async () => {
```

Comment:
This is the only test failing currently because it can't find a mock for the /operating_system request:
```
Could not find mock for: 

    {

      "method": "get",

      "url": "/api/inventory/v1/system_profile/operating_system"

    }
```
Still don't know why all other tests pass with the same mocks configuration.